src/main/resources/scripts/rwa-2903_update_civil_role_categories_work_types.sql

```diff
@@ -0,0 +1,51 @@
+--Pre Deployment Data preparations - volumes obtained 07/05/24
+select count(*) from cft_task_db.tasks where role_category is null and jurisdiction = 'CIVIL';
+
+--count|
+-------+
+--    2
+
+--  take a backup of all the task_id's into a CSV file OR create a temporary table as below
+CREATE TABLE tmp_task_ids_for_civil_role_category AS SELECT task_id FROM cft_task_db.tasks WHERE role_category is null and jurisdiction = 'CIVIL';
+
+select count(*), task_type  from cft_task_db.tasks where role_category is null and jurisdiction = 'CIVIL' group by task_type;
+
+--count|task_type                          |
+-------+-----------------------------------+
+--    1|FastTrackDirections                |
+--    1|reviewSpecificAccessRequestLegalOps|
+
+select count(*) from cft_task_db.tasks where work_type is null and jurisdiction = 'CIVIL';
+
+--count|
+-------+
+--   24
+CREATE TABLE tmp_task_ids_for_civil_work_type AS SELECT task_id FROM cft_task_db.tasks WHERE work_type is null and jurisdiction = 'CIVIL';
+
+select count(*), task_type  from cft_task_db.tasks where work_type is null and jurisdiction = 'CIVIL'  group by task_type ;
+
+--count|task_type                        |
+-------+---------------------------------+
+--    3|FastTrackDirections              |
+--    1|LegalAdvisorRevisitApplication   |
+--    7|transferCaseOffline              |
+--   13|transferCaseOfflineNotSuitableSDO|
+
+-- IMPLEMENTATION STEPS
+-----------------------------
+update cft_task_db.tasks set role_category = 'JUDICIAL' where role_category is null and jurisdiction = 'CIVIL'  and task_type in ('FastTrackDirections');
+update cft_task_db.tasks set role_category = 'LEGAL_OPERATIONS' where role_category is null and jurisdiction = 'CIVIL' and task_type in ('reviewSpecificAccessRequestLegalOps');
+
+update cft_task_db.tasks set work_type = 'decision_making_work' where work_type is null and jurisdiction = 'CIVIL' and task_type in ('FastTrackDirections','LegalAdvisorRevisitApplication');
```

Comment:
I assume we've checked `LegalAdvisorRevisitApplication` should be decision_making_work with Civil?

Reply:
Yes, we checked with Civil team